### PR TITLE
VSCode Entrypoint & Theme

### PR DESCRIPTION
**Pull request by JacobMGEvans**

### CODE REVIEW

File: packages/quick-edit-extension/src/extension.ts
Comment (penalosa) [inline packages/quick-edit-extension/src/extension.ts:1]: Does this apply any changes to the current entrypoint? (bolding etc...)?

Comment (JacobMGEvans) [reply packages/quick-edit-extension/src/extension.ts:1]: No, that was moved into another ticket. Based on a conversation on Monday with @lrapoport-cf & @admah

File: package-lock.json [hunk 1]
```diff
@@ -44076,6 +44080,419 @@
 				"url": "https://github.com/sponsors/wooorm"
 			}
 		},
+		"packages/cloudflare-logic": {
```
[the remark this answers is not in the document]
Comment (JacobMGEvans) [reply package-lock.json:44083]: Got it removed, updating the Lock again.

File: package-lock.json [hunk 2]
```diff
@@ -44076,6 +44076,18 @@
...
+			"extraneous": true,
+			"devDependencies": {
+				"@types/vscode": "^1.76.0",
+				"esbuild": "^0.17.11",
+				"typescript": "^4.9.5"
+			},
+			"engines": {
+				"vscode": "^1.76.0"
+			}
+		},
```
Comment (penalosa) [inline package-lock.json:44090]: This package still doesn't exist :(

**Review by penalosa — APPROVED** [review]
Other than the lock file issue, LGTM!

**Pull request closed without merging**